File: Iniciante/1049.c
#include <stdio.h>

int main(){

    char class1[13], class2[9], class3[11];

    scanf("%s", class1);
    scanf("%s", class2);
    scanf("%s", class3);

    if(class1[0] == 'v'){
        if(class2[0] == 'a'){
            if(class3[0] == 'c'){
                printf("aguia\n");
            }else printf("pomba\n");

        }else if(class3[0] == 'o'){
            printf("homem\n");
        }else printf("vaca\n");

    }else if(class2[0] == 'i'){
        if(class3[2] == 'm'){
            printf("pulga\n");
        }else printf("lagarta\n");

    }else if(class3[0] == 'h'){
        printf("sanguessuga\n");
    }else printf("minhoca\n");

    return 0;

}

File: Iniciante/1049.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        String class1 = scanf.nextLine();
        String class2 = scanf.nextLine();
        String class3 = scanf.nextLine();

        if(class1.intern() == "vertebrado"){
            if(class2.intern() == "ave"){
                if(class3.intern() == "carnivoro"){
                    System.out.println("aguia");
                }else System.out.println("pomba");
    
            }else if(class3.intern() == "onivoro"){
                System.out.println("homem");
            }else System.out.println("vaca");
    
        }else if(class2.intern() == "inseto"){
            if(class3.intern() == "hematofago"){
                System.out.println("pulga");
            }else System.out.println("lagarta");
    
        }else if(class3.intern() == "hematofago"){
            System.out.println("sanguessuga");
        }else System.out.println("minhoca");

    }
}